run_dir/design/worksets.html

```diff
@@ -73,18 +73,39 @@ <h2 id="{{ worksets }}" class="worksets_page_heading"> <small>Worksets from the
                          {% raw '</samp><hr style="margin:0;"><samp>'.join(ws_data[onews].get(onekey[1])) %}
                          </samp></td>
                      {% elif onekey[1] in ['passed', 'failed', 'unknown', 'total'] %}
-                         <td>{{ ws_data[onews].get('samples', {}).get(onekey[1]) }}</td>
+                         <td>{{ ws_data[onews].get('samples', {}).get(onekey[1]) }}</td> 
+                     {% elif onekey[1] == 'Workset Notes'%}
+                         <td class="latest_workset_note" class="text-center" style="min-width:250px">
+                         {% if ws_data[onews].get('Workset Notes') %}
+                              {% set workset_notes = ws_data[onews].get('Workset Notes') %}
+                              {% set latest_workset_note_key = list(workset_notes.keys())[0] %}
+                              <div class="panel panel-default running-note-panel">
+                              <div class='panel-heading'>
+                              <a href='mailto:{{ workset_notes[latest_workset_note_key]["email"].encode("ascii","ignore")}}'>{{ workset_notes[latest_workset_note_key]["user"].encode("ascii","ignore")}}</a>  -  {{ datetime.datetime.strptime(latest_workset_note_key, "%Y-%m-%d %H:%M:%S.%f").strftime("%a %b %d %Y, %I:%M:%S %p") }}
+                              </div>
+                              <div class='panel-body' style='white-space: pre-line'>
+                              {{workset_notes[latest_workset_note_key]['note']}}
+                              </div>
+                              </div>
+                         {% end %}
+                        </td>
                      {% else %}
                         <td>{{ ws_data[onews].get(onekey[1]) }}</td>
                      {% end %}
                  {% end %}
-                 </tr>
                  {% end %}
+                </tr>
             </tbody>
           </table>
         </div>
       </div>
-
+      <script type="text/javascript">
```

Comment:
Is make_markdown called on the panel body twice, once here and once in load_table() in worksets.js?

Reply:
It is! Hmmm....I only added it to the HTML-file because when I inspected in the browser, the markdown class wasn't applied to the workset note as it was to the running note.....I can check again!

Reply:
Yeah, it's not showing the markdown:
<img width="530" alt="Skärmavbild 2019-12-06 kl  13 43 34" src="https://user-images.githubusercontent.com/37932118/70324012-a42a0180-182e-11ea-90c8-92468268d3e9.png">
When I added the extra lines of code in the HTML:
<img width="528" alt="Skärmavbild 2019-12-06 kl  13 40 30" src="https://user-images.githubusercontent.com/37932118/70324034-b73cd180-182e-11ea-99ad-91dc229256ec.png">